pom.xml

```diff
@@ -23,10 +27,119 @@
         <developerConnection>scm:git@github.com:WASdev/sample.swagger.git</developerConnection>
         <url>git@github.com:WASdev/sample.swagger.git</url>
     </scm>
+    
+	<dependencies>
+		<dependency>
+			<groupId>javax.json</groupId>
+			<artifactId>javax.json-api</artifactId>
+			<version>1.0</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>javax.ws.rs</groupId>
+			<artifactId>javax.ws.rs-api</artifactId>
+			<version>2.0</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>io.swagger</groupId>
+			<artifactId>swagger-annotations</artifactId>
+			<scope>provided</scope>
+			<version>1.5.3</version>
+		</dependency>
+		<dependency>
+			<groupId>javax.servlet</groupId>
+			<artifactId>javax.servlet-api</artifactId>
+			<version>3.1.0</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.12</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.cxf</groupId>
+			<artifactId>cxf-rt-rs-client</artifactId>
+			<version>3.1.1</version>
+			<scope>test</scope>
+		</dependency>
+	</dependencies>
 
-    <modules>
-        <module>swagger-sample-application</module>
-        <module>swagger-sample-wlpcfg</module>
-    </modules>
+	<properties>
+		 <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<!-- Update this to match the port number in the swaggerSample server.xml 
+			This is needed to run the tests -->
+		<libertyTestPort>9081</libertyTestPort>
+		<skipFVT>false</skipFVT>
```

Comment:
I agree you can remove this from the sample.  Add to the documentation that if they can use `skipTests` to skip testing.  http://maven.apache.org/surefire-archives/surefire-2.17/maven-failsafe-plugin/examples/skipping-test.html